pandas/tests/frame/methods/test_set_axis.py

```diff
@@ -58,13 +58,22 @@ def test_set_axis_copy(self, obj):
         tm.assert_equal(expected, result)
         assert result is not obj
         # check we DID make a copy
```

Comment:
this comment feels a bit out of place now, perhaps move it after the `if` statement (and then an opposite one in the `else`)?